docs/publishing/sections/5_analytics_portfolio_site.md

```diff
@@ -206,3 +209,9 @@ build_my_reports:
     git add portfolio/my_report/district_*/ portfolio/my_report/*.yml portfolio/my_report/*.md
     git add portfolio/sites/my_report.yml
 ```
+
+### Delete Portfolio/ Refresh Index Page
```

Comment:
I'm rereading this again and can you ask Tiffany or Eric how to take off a site analysis.calitp.org completely? These instructions are about redeploying your portfolio, rather than deleting a portfolio completely.